shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -783,6 +815,27 @@ protected void onFirstFrameRendered() {}
      * to provide its own {@code FlutterEngine} instance.
```

Comment:
comment for line 811

> `Returns the {@link FlutterEngine} that should be used by a child {@code FlutterFragment}.` 

This is just an interface right? We don't know that it has a child per se.

Reply:
It's an interface that is defined within `FlutterFragment` for use by `Activity`s that contain `FlutterFragment`s.

Reply:
Talked offline, resolving old comments.